docs/execution_nodes.md

```diff
@@ -0,0 +1,83 @@
+# Adding execution nodes to AWX
+
+Stand-alone execution nodes can be added to run alongside the Kubernetes deployment of AWX. These machines will not be a part of the AWX Kubernetes cluster. The control nodes running in the cluster will connect and submit work to these machines via Receptor. The machines be registered in AWX as type "execution" instances, meaning they will only be used to run AWX Jobs (i.e. they will not dispatch work or handle web requests as control nodes do).
+
+Below is an example of a single AWX pod connecting to two different execution nodes. For each execution node, the awx-ee container makes an outbound TCP connection to the machine via Receptor.
+
+```
+                                 AWX POD
+                             ┌──────────────┐
+                             │              │
+                             │ ┌──────────┐ │
+┌─────────────────┐          │ │ awx-task │ │
+│ execution node 1│◄────┐    │ ├──────────┤ │
+├─────────────────┤     ├────┼─┤ awx-ee   │ │
+│ execution node 2│◄────┘    │ ├──────────┤ │
+└─────────────────┘ Receptor │ │ awx-web  │ │
+                      TCP    │ └──────────┘ │
+                     Peers   │              │
+                             └──────────────┘
+```
+
+Note, if the AWX deployment is scaled up, the new AWX pod will also make TCP connections to each execution node.
+
+
+## Overview
+Adding an execution instance involves a handful of steps:
+
+1. Start a machine that is accessible from the k8s cluster (Red Hat family of operating systems are supported)
+2. Create a new AWX Instance with `hostname` being the IP or DNS name of your remote machine.
+3. Download the install bundle for this newly created instance.
+4. Enable upstream Receptor repository on the machine.
```

Comment:
we can add link to copr repo here https://copr.fedorainfracloud.org/coprs/ansible-awx/receptor/